packages/vscode-extension/src/handlers.ts

```diff
@@ -1944,15 +1948,68 @@ export async function openAdaptiveCardExt(
       "Cancel"
     );
     if (selection === "Install") {
+      ExtTelemetry.sendTelemetryEvent(
+        TelemetryEvent.AdaptiveCardPreviewerInstallConfirm,
+        getTriggerFromProperty(args)
+      );
       await vscode.commands.executeCommand("workbench.extensions.installExtension", acExtId);
-      await vscode.commands.executeCommand("workbench.view.extension.cardLists");
+    } else {
+      ExtTelemetry.sendTelemetryEvent(
+        TelemetryEvent.AdaptiveCardPreviewerInstallCancel,
+        getTriggerFromProperty(args)
+      );
     }
-  } else {
```

Comment:
What's the behavior when user has already installed the extension and clicks "Preview and Debug Adaptive Cards" in tree view?

Reply:
If already installed, nothing will happen. Because old adaptive card studio has treeview, it would open treeview previously. But new adaptive card preview extension doesn't have treeview.

Reply:
<img width="868" alt="image" src="https://github.com/OfficeDev/TeamsFx/assets/9698542/9ae46630-89f4-44e6-9ff8-34fca6067a2d">
Added a message to tell user how to preview card if already installed.